src/appComponents/tracks/TracksController.js

```diff
@@ -1,100 +1,121 @@
 /**
  * Created by championswimmer on 29/5/15.
  */
+(function() {
+    "use strict";
 
-var tracksModule = angular.module('oe.tracks', ['ui.router']);
+const tracksModule = angular.module("oe.tracks", ["ui.router"]);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["angular" is not defined.](https://www.codacy.com/app/triplez/open-event-webapp/file/2419561640/issues/source?bid=3233857&fileBranchId=3238420#l7)